include/eve/module/core/regular/replace.hpp

```diff
@@ -15,56 +15,54 @@
 
 namespace eve
 {
-// TODO DOC is it to be in user interface ?
-//================================================================================================
-//! @addtogroup core_logical
-//! @{
-//!    @var replace_ignored
-//!    @brief A small helper to replace ignored values
-//!
-//!   @groupheader{Header file}
-//!   @code
-//!   #include <eve/module/core.hpp>
-//!   @endcode
-//!
-//!   A convinience wrapper around if_else.
-//!
-//!   @groupheader{Callable Signatures}
-//!
-//!   @code
-//!   namespace eve
-//!   {
-//!      template< eve::value T, eve::conditional_expr Ignore, eve::value Other >
-//!      auto replace_ignored(T x, Ignore ignore, Other with )
-//!        -> decltype(eve::if_else(ignore, x, with))
-//!   }
-//!   @endcode
-//!
-//!   **Parameters**
-//!
-//!   * `x` - main value
-//!   * `ignore` - selection
-//!   * `with` - value to replace with
-//!
-//!   **Return value**
-//!
-//!   Same as if_else.
-//!
-//================================================================================================
-
-EVE_MAKE_CALLABLE(replace_ignored_, replace_ignored);
+  template<typename Options>
+  struct replace_ignored_t : callable<replace_ignored_t, Options>
+  {
+    template<simd_value Wide, conditional_expr Ignore, typename Other>
```

Comment:
No to typename. In the new format we always specify the interfaces fully.

Now that I think about it - we should probably accept `eve::zero` and other things like that here. Can you maybe make that work?